src/test/resources/uk/gov/justice/digital/hmpps/hmppstier/integration/bdd/OGRS.feature

```diff
@@ -0,0 +1,72 @@
+#lang: en
+Feature: OGRS
+
+  Scenario: Take 10's less than 10%
```

Comment:
I don't understand the scenario title here. If it means something to the business that's not a problem though

Reply:
Its from Stephan, I  can rephrase to "Ogrs score of xxx scores x points"?